### PR TITLE
✨ Add convertLongs

### PR DESCRIPTION
**Pull request by ghost**
Aid users who are struggling with Long.js objects. Feedback appreciated.

### CODE REVIEW

**Comment by cyraxx** [comment]
We can have this in Utils for sure. The only thing I don't like are the unnecessary parantheses around the ternary operators but that's just personal preference 😛 

I'd even go as far as suggesting this could be done automatically on every response (with an option to turn it off) because when do you need the `Long`s really? Any comments on that?

**Comment by Alph44rchitect** [comment]
I like the idea of @cyraxx 

> I'd even go as far as suggesting this could be done automatically on every response (with an option to turn it off) because when do you need the Longs really? Any comments on that?

Cause of less duplicated code and work, that no one needs.

File: pogobuf/pogobuf.client.js [hunk 1]
```diff
@@ -163,6 +163,15 @@ function Client() {
     this.setResponseCallback = function(callback) {
         self.on('raw-response', callback);
     };
+    
+    /**
+     * Enables or disables automatic Long.js object conversion in RPC responses
+     * @param {boolean} enable
+     */
+    this.setAutomaticLongConvertingEnabled = function(enable) {
```
Comment (cyraxx) [inline pogobuf/pogobuf.client.js]: This should be conversion not converting I think.

File: pogobuf/pogobuf.client.js [hunk 2]
```diff
@@ -163,6 +163,15 @@ function Client() {
     this.setResponseCallback = function(callback) {
         self.on('raw-response', callback);
     };
+    
+    /**
+     * Enables or disables automatic Long.js object conversion in RPC responses
```
Comment (cyraxx) [inline pogobuf/pogobuf.client.js]: Maybe expand a little: Enables or disables automatic conversion of Long.js objects to primitive types in API response objects.

**Pull request merged**